src/please.go

```diff
@@ -509,7 +509,7 @@ var buildFunctions = map[string]func() int{
 		if !success {
 			return toExitCode(success, state)
 		}
-		return debug.Debug(state, opts.Debug.Args.Target, opts.Debug.Args.Args)
+		return debug.Debug(state, opts.Debug.Args.Target, opts.Debug.Args.Args, core.RepoRoot)
```

Comment:
`core.RepoRoot` would be the default run location but this could be easily extended by making a `--wd` flag available if required.